GCTC-NTGC/gc-digital-talent
# Feature - 2765 -  Replace skills filters

Resolves #2765 

## Summary

This replaces the CMO Assets checkboxes with Skills on the `/talent/search` page.

## Testing

1. Navigate to `/talent/search`
2. Scroll down to "Skills as filters"
3. Add a few skills
4. Ensure a query occurs and proper results are returned
